### ISSUE TITLE
Utilisateur / Mon compte

### ISSUE THREAD

**Issue by webaaz** [afup/back-office#3]
On garde les fonctionnalités à ISO.
Dans l’interface, on ajoute les champs
- compte github
- compte twitter
- tags

**Comment by jewome62** [comment]
Que sont les tags ?
La même chose que sur badge du Forum PHP ???

**Comment by webaaz** [comment]
Oui, ce n'est pas le truc le plus important. Ça permet de tagger les gens
par techno ou de retrouver les membres des bureaux précédents...
